client/src/containers/mealList.js

```diff
@@ -3,37 +3,49 @@ import { connect } from 'react-redux';
 import { fetchMeals } from '../actions/index';
 import { Link } from 'react-router-dom';
 import _ from 'lodash';
+import { GridList, GridTile } from 'material-ui/GridList';
+import IconButton from 'material-ui/IconButton';
+import Subheader from 'material-ui/Subheader';
+import StarBorder from 'material-ui/svg-icons/toggle/star-border';
+
+const styles = {
```

Comment:
whats cool is if i move this css to another file, it actually changes the display of the meals from squares into longer rectangles (which i actually like better), will show you guys tomorrow and you guys can pick which style you like